### DISCUSSION TITLE
How do you show the time each test can take?

### DISCUSSION THREAD

**Discussion by matthew-dean** [vitest-dev/vitest#4010] [Q&A]
I feel like an idiot. I've tried every trick I can think of, but I can't get the time to show per test. It shows for a `describe` or `it` block if it's over a certain threshold, but I can't figure out how to configure that.

Regardless, I don't want a yellow warning, I want a plain (or green) output of the total test time (or yellow if it's over a certain threshold).

I've tried defining the tests differently. I've tried console logging (which definitely doesn't work). I've searched all the APIs for SOME WAY to output something next to the test description. Or change any kind of logging? Or put anything in the console in a predictable fashion? I've tried configuring reporters (which definitely don't seem to do what the documentation says it does? -- or they do and I don't understand what it's saying?)

How on earth can you LOG HOW MUCH TIME A TEST TOOK NEXT TO EACH TEST DESCRIPTION?

At the very least, _how do you log something under the description?_ I've torn all my hair out. I have no hair.

**Comment by exoego** [comment]
https://vitest.dev/config/#slowtestthreshold

- Config: `slowTestThreshold: <number>`
- CLI: `--slow-test-threshold=<number>` or `--slowTestThreshold=<number>`

It looks like we can't output total test time in green, only in yellow with `slowTestThreshold: 0`

**Comment by spamshaker** [comment]
maybe one day...
https://github.com/vitest-dev/vitest/pull/6432
